internal/vpn/os_client_darwin.go

```diff
@@ -0,0 +1,38 @@
+//+build darwin
+
+package vpn
+
+import (
+	"bytes"
+	"fmt"
+	"net"
+	"os/exec"
+)
+
+const (
+	defaultNetworkGatewayCMD = "netstat -rn | sed -n '/Internet/,/Internet6/p' | grep default | awk '{print $2}'"
```

Comment:
I would use **`** instead of **"** quotes to encapsulate the string. Might bring a better reading experience. Again, this is personal taste.

Reply:
This comment is really for other `os_client_*.go` files with the same variable that resorts to using backslashes.

Reply:
it would've bring more readability if there were some some escaped quotes there, so we could remove escaping. but in this case it changes nothing, literally. but I applied this in `os_client_linux`, there were escaped quotes there. so thank you for this anyway